External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -1200,17 +1195,12 @@ void OpDispatchBuilder::SHRDOp(OpcodeArgs) {
   else
     Shift = _And(Shift, _Constant(0x1F));
```

Comment:
What about size==16? don't we need extra masking there?

Reply:
Nope. x86-64 masks 0x3F for 64bit ops and 0x1F for anything lower.
This allows you to shift things out of the register in those instances

Reply:
Oh okay. Such consistency much x86...